data/companies.yml

```
@@ -20,7 +20,7 @@ providers:
     logo: "/logos/tetrate.svg"
     url: "https://www.tetrate.io/tetrate-service-bridge/"
   - name: "Gloo Mesh by Solo.io"
-    logo: "/logos/solo.png"
+    logo: "/logos/gloomesh.svg"
```

```suggestion
    logo: "/logos/solo.svg"
```

The companies should use the company's logo

---

That is intentional because this is gloo mesh in the context of the page.  Not solo.io.

---

Under the `providers` page we use the company's logo rather than a product logo: https://deploy-preview-15894--preliminary-istio.netlify.app/latest/about/ecosystem

---

Ok, i see that now, a bit confusing as name is gloo mesh. :). pushed a change for it.